ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -132,6 +132,11 @@ const WorkflowNodeSummary = (props: Props) => {
         });
     }
 
+    const object = props.workflow;
```

Comment:
I think we should have an object that is like the node status really.

```
{
                            metadata: {
                                namespace: props.workflow.metadata.namespace,
                                name: props.node.id
                            },
                            workflow: workflow,
                            status: {
                                startedAt: props.node.startedAt,
                                finishedAt: props.node.finishedAt
                            }
                        }}
```

That way you can reference the workflow name, and its started and finished at times.